### ISSUE TITLE
Wish: allow importing of comment along with IP address

### ISSUE THREAD

**Issue by simonmcconnell** [angryip/ipscan#174]
It would be nice if I could setup a txt file as such:
```
IP, COMMENTS
192.168.0.1, COMPUTERX
192.168.0.2, SW9023
192.168.0.3, COMPUTERY
192.168.0.4, Motor Controller
```
so that the Comments column is filled with the data provided when the text file is loaded.  In my use case I would have dozens of text files, one for each area and piece of equipment so that users could quickly see if all the relevant ethernet devices are online.  They are not going to know what IP is what without another reference.

**Comment by cvctech** [comment]
Yes please! I have a number of IP addresses for nodes across our campus, and when someone says "the internet isn't working" I like to be able to load a txt file and ping them all and see what (if anything) doesn't respond. 

Similarly, I'm grateful that a txt file populated with FQDN instead of IP addresses resolves to IP addresses, but it would be handy if those original domains would show up under the hostname or comment column.

**Comment by oapy123** [comment]
I too would like this feature
